ush/config_defaults.yaml

```diff
@@ -245,8 +245,8 @@ platform:
   # RUN_CMD_SERIAL:
   # The run command for some serial jobs 
   #
-  # RUN_CMD_AQM:
-  # The run command for some AQM tasks.
+  # RUN_CMD_NEXUS:
+  # The run command for the AQM NEXUS tasks.
```

Comment:
Please update the variable name in `doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst`.  Please add `FIXaqm` and `FIXemis` entries to `Fixed File Directory Parameters` section of that file as well.

Reply:
Updated.

Reply:
Thanks, @chan-hoo!